app/scripts/models/auth_brokers/fx-ios-v1.js

```diff
@@ -26,22 +27,13 @@ define(function (require, exports, module) {
     _notifyRelierOfLogin: function (account) {
       /**
        * As a workaround for sign-in/sign-up confirmation view disappearing
-       * on iOS, delay the login message sent via the channel by 5 seconds.
+       * on iOS, delay the login message sent via the channel by LOGIN_MESSAGE_DELAY_MS.
        * This will give the user an indication that they need to verify
        * their email address.
        */
-      const defer = p.defer();
-
-      this.window.setTimeout(() => {
-        var loginData = this._getLoginData(account);
-
-        if (! this._hasRequiredLoginFields(loginData)) {
-          defer.resolve();
-        }
-        this.send(this.getCommand('LOGIN'), loginData);
-      }, 5000);
-
-      return defer.promise;
+      return p().delay(Constants.LOGIN_MESSAGE_DELAY_MS).then(() => {
```

Comment:
Yeah, when we initialize the broker we could pass in a `loginMessageDelayMS` option, if it's set, use it, if not, default to Constants.LOGIN_MESSAGE_DELAY_MS.

Could you rename `LOGIN_MESSAGE_DELAY_MS` to something a bit more specific since it's only used for iOS v1 at this point? Maybe `IOS_V1_LOGIN_MESSAGE_DELAY_MS`?